sg.tf

```diff
@@ -0,0 +1,65 @@
+resource "aws_security_group" "sg" {
+  name        = "${var.project}-${var.environment}-${var.name}"
```

Comment:
can you use `name_prefix`? That way you make sure you don't have name collisions and it's more future-proof imo

Reply:
Do we really like `name_prefix`?

Reply:
I prefer prefix as well. its always unique and not too long

Reply:
ok fair enough. My only concern was about how meaningful the name will become but I guess that with tags that's fixed

Reply:
Done

Reply:
yes indeed, we normally have the tags to identify and differentiate resources, and we should keep names as generic as possible imo